### PR TITLE
App store Api test cases

### PR DESCRIPTION
**Pull request by nidhiya-pious**

### CODE REVIEW

File: cypress/e2e/AppStore/apiInfo.cy.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to click on the "API info" tab', () => {
+        cy.get('#details-tab > .d-md-flex > span').click();
+    });
+
+    it('3. Test to check for the title "Usage"', () => {
+        cy.checkTextVisibility('Usage');
+    });
```
Comment (abhishek9celestial) [inline cypress/e2e/AppStore/apiInfo.cy.js:20]: Can you check any two random endpoints in the Usage list.

File: cypress/e2e/AppStore/documentation.cy.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import { APP_STORE_BASE_URL} from '../../resources/data';
+
+describe('App store documentation page', () => {
```
Comment (abhishek9celestial) [inline cypress/e2e/AppStore/documentation.cy.js]: make it as Documentation.

File: cypress/e2e/AppStore/documentation.cy.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+    it('2. Test to click on the "Documentation" tab', () => {
+        cy.get('#documentation-tab').click();
+        cy.checkTextVisibility('App Store API Reference');
+    });
+
+    it('3. Test to check and click link "development quickstart guide."', () => {
+        cy.contains('a', 'development quickstart guide').should('be.visible');
+        cy.contains('a', 'development quickstart guide').click();
+    });
```
Comment (abhishek9celestial) [inline cypress/e2e/AppStore/documentation.cy.js:24]: How you are navigating back to the App store page after clicking the link.

File: cypress/e2e/AppStore/documentation.cy.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+    it('12. Test to click on the "Error Codes" link', () => {
+        cy.get('li > a').contains('Error Codes').click();
+        cy.get('[name="errors"]').should('be.visible');
+    });
+
+    it('13. Test to click on "Accounts Pages" link', () => {
+        cy.contains('a', 'Accounts page').should('be.visible');
+        cy.contains('a', 'Accounts page').click();
+     });
```
Comment (abhishek9celestial) [inline cypress/e2e/AppStore/documentation.cy.js:71]: Here also how you're navigating back to the current page.

File: cypress/e2e/AppStore/home.cy.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        cy.get('.sidebar-content', { timeout: 7000 }).should('be.visible');
+        cy.get('body').click();
+    });
+
+    it('6. Test to check on "Code example" window is present', () => {
+        cy.componentVisiblityCheck('.code-response');
+    });
+
+    it('7. Test to click link "check the documentation."', () => {
+        cy.get('a').contains('check the documentation.').click();
```
Comment (abhishek9celestial) [inline cypress/e2e/AppStore/home.cy.js:38]: same here.

Comment (nidhiya-pious) [reply cypress/e2e/AppStore/home.cy.js:38]: This is changing the tab to Documentation. So no need to go back to the base url

**Pull request merged**